pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1670,6 +1670,54 @@ func (n *NetworkPolicyController) cleanupOrphanGroups(internalNetworkPolicy *ant
 	}
 }
 
+// verifyPoliciesProcessed checks that all the policy objects in the cluster supported by Antrea
+// are already processed by the Antrea controller. It serves as a sanity check/prerequisite for
+// the networkpolicyanalysis command, since the command uses controller cache as computation
+// source. Note that:
+//  1. This function blocks internal NP processing until it returns, which is acceptable given
+//     the rarity of invocation.
+//  2. Verification does not guarantee the latest version of the policy is processed.
```

Comment:
Probably add more details here to help understanding? Is it the original policy, does latest version mean update, is this caused by async?

Reply:
How about deletion? If I understand this "guarantee", verification also depends on the fact that all older versions of the policies/removed policies do not exist (which is indeed true).

Reply:
Done